Makefile

```diff
@@ -241,6 +267,7 @@ openstack_deploy_prep: openstack_deploy_cleanup ## prepares the CR to install th
 	pushd ${OPERATOR_BASE_DIR} && git clone -b ${OPENSTACK_BRANCH} ${OPENSTACK_REPO} && popd
 	cp ${OPENSTACK_CR} ${DEPLOY_DIR}
 	bash scripts/gen-service-kustomize.sh
+	bash scripts/gen-service-container-kustomize.sh
```

Comment:
One thing I would point out is that calling `gen-service-container-kustomize.sh` after `gen-service-kustomize.sh` will overwrite the `kustomization.yaml` from the former.  That YAML was already consumed and is not technically needed anymore, but perhaps someone might want to look at its contents for debugging purposes after the fact.  There are a few different ways to avoid this if we care here, but I just thought I'd mention it.

Reply:
Thank you @abays for pointing out, yes kustomization.yaml is getting overwritten. I agree with you to keep the content for debugging purpose. Please suggest the changes in order to avoid this.

Reply:
One suggestion might be to merge `gen-service-container-kustomize.sh` into `gen-service-kustomize.sh`, unless you think there might be scenarios where we would be executing the logic in one but would not necessarily want to use the logic of the other.  I think it makes semantic sense to have both kustomization processes under one `gen-service-kustomize.sh` script, because it's all ultimately targeting the OpenStack service we're deploying.

Reply:
That sounds good. I have made the changes. Please have a look.

Reply:
That sounds good. I have made the changes. Please have a look.